README.md

```diff
@@ -4,11 +4,11 @@ envvar is a tiny JavaScript package for deriving JavaScript values from
 environment variables.
 
 ```javascript
-var envvar = require('envvar');
+const envvar = require('envvar');
 
-var GITHUB_API_TOKEN = envvar.string('GITHUB_API_TOKEN');
-var HTTP_MAX_SOCKETS = envvar.number('HTTP_MAX_SOCKETS');
-var ENABLE_FEATURE_X = envvar.boolean('ENABLE_FEATURE_X', false);
+const GITHUB_API_TOKEN = envvar.string('GITHUB_API_TOKEN');
+const HTTP_MAX_SOCKETS = envvar.number('HTTP_MAX_SOCKETS');
+const ENABLE_FEATURE_X = envvar.boolean('ENABLE_FEATURE_X', false);
```

Comment:
So long 2014!

It's pleasing that we did not make a single change to this project in 2015. Some projects actually can be "finished". :)